draft-sharabayko-srt.md

```diff
@@ -1225,9 +1225,9 @@ ACKACK control packets do not contain Control Information Field (CIF).
 ### Message Drop Request {#ctrl-pkt-dropreq}
 
 A Message Drop Request control packet is sent by the sender to the receiver
-when it requests the retransmission of an unacknowledged packet (all or part
-of a message) which is not present in the sender's buffer. This may happen, for
-example, when a TTL parameter (passed in the sending function) triggers a
+when a retransmission of an unacknowledged packet (forming a whole or a part
+of a message) which is not present in the sender's buffer was requested.
+This may happen, for example, when a TTL parameter (passed in the sending function) triggers a
 timeout for retransmitting lost packets which constitute parts of a message, causing
 these packets to be removed from the sender's buffer.
```

Comment:
```suggestion
when a retransmission of an unacknowledged packet (forming a whole or a part
of a message) which is not present in the sender's buffer is requested.
This may happen, for example, when a TTL parameter (passed in the sending function) triggers a
timeout for retransmitting one or more lost packets which constitute parts of a message, causing
these packets to be removed from the sender's buffer.
```